docs/services/proxy/_index.md

```diff
@@ -7,4 +7,10 @@ geekdocFilePath: _index.md
 geekdocCollapseSection: true
 ---
 
+## Abstract
+
 This service provides a proxy service that routes requests to the correct extensions.
```

Comment:
```suggestion

The proxy service acts as an API Gateway and routes requests to the correct target service. It also provides standard proxy services.
```
This text is also used in our documentation and was discussed with butonic. Either commit or suggest changes which we then need to transfer to the documentation.